keyboards/eniigmakeyboards/ek65/readme.md

```diff
@@ -0,0 +1,17 @@
+# Eniigma Keyboards ek65
+
+![ek65](https://i.imgur.com/s6DI5vu.png)
+
+A 65% PCB with symmetrical bottom row layout, JST daughterboard support, designed for use with Eniigma Keyboards products.
+
+![layout](https://agora-file-storage-prod.s3.us-west-1.amazonaws.com/wms-offer-attachments/5646239540705846621?response-content-disposition=inline%3B%20filename%3D%22ek65.png%22%3B%20filename%2A%3Dutf-8%27%27ek65.png&X-Amz-Security-Token=IQoJb3JpZ2luX2VjEAYaCXVzLXdlc3QtMSJHMEUCIEdF9OlP%2BQaSM0ZzWJOv3REu80CvNyNQfwFYqRHR4tjUAiEArQp4I1ISLiD3xB7n0AUtr2Wr2cE20%2FXALReN2%2BdFlCoqtwQIv%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2FARAAGgw3Mzk5MzkxNzM4MTkiDHkcmRcj%2F21IPAqXgCqLBGPdZ5smJ61izHPoSE%2F4Aawe%2FHu%2BBQlzDUw0Knk2p5oxtXP0bb2s6Itp7tswp38LczP87foyDkKaU2uHuiuGdS0m43JHMPwd0WHEpV6p%2BdGsbOso7KJxYtj33Asi6k2iy8SdjCjUgFgOIoqBs9tioqYuAqptQRIAoOL%2Bd6uEnDQpcCH2jXzrvX8bd1g60mA%2FtAoIVPB3wzkcj7QKvzwYDRXJZ6uiHZWEc8gd35PBSUIlAtBiUIPyRzStGJLk6T9PKFO8F7Ou69t5fiq93Y3dBkCDnMq1VkX3Wg4feJPriKLjziPD5cjwr9ftpkkdj6n%2Fi08nDtz27HRuWUAonn1hyzPq6W3xgKHl9TGzQGGJiBGjB%2BpXRY9ghpOXnmIB7vfMmSn6r9iDWBG6pgmMyK4ktN2Xy16y0KrX%2B%2Be%2B%2BNqzc%2BGezQSfVXxofJF%2Be4nx2aKGexASN4HO7lGXhGvK7F0TP3tKgU82Mb8CqF1yJpN95IT%2BfDPbPN9BhbFRDTAcsvWBKLzVuyLxzMhqDpWGXnOk44DMupdl0y0aTxvGIsUhFVSe70PJ3g5MYfnZeoDsmTxPlp8YERS4QpK%2Bhq6OEWBoByHR1YmgDsj8rC53MaNeemureVzYIBM5BEIVy3Aiv1%2Blewm7pmlh3wWh6A%2Ft%2B9DEdHii6i0XAeO34cEAHGp%2Fdvs23k7XwsSOeRtK8fIww6D2%2BAU68AHWI0a%2Fa08hd5%2BD4GfymWLnByZPakg3BEvIBtQdDL4JZYx0pPldITdO2vpaKR5RlQPSOFXnFT9FiIui5NLwn1GXGAo9V%2FHUM4Um6FisHoaxrBOfMz6J83%2FGP3tuvJs63fVrHhxcdiQ6PulzGTwRVysvdkOp2Y%2BlXGIefkNd1sxWf%2FxWLZWuiU%2FkaI85Pl1%2B20eXR0WlbA9%2FdJsEipXLZ%2FUxuLa%2Fb5sO5xV2NXcjbLs3SvEAXtRLc6edVWO6XrvyWbMt1mQeB2FuQ1nDUNFctmOsHkvLzBdjYv5I%2FvXq0wisoAQ0hnR5B4sDCDf4LdBWtdw%3D&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Date=20200726T145009Z&X-Amz-SignedHeaders=host&X-Amz-Expires=900&X-Amz-Credential=ASIA2YR6PYW5TDEFAM5N%2F20200726%2Fus-west-1%2Fs3%2Faws4_request&X-Amz-Signature=8f36292ceb56d95b297dc675d45069f0e8535a6eb1c29b7b74a6bf38cea8098d)
```

Comment:
This layout image doesn't work. Should rehost this on someplace like Imgur.

Reply:
We already fixed this with a commit a little while ago (https://i.imgur.com/s6DI5vu.png). I'll take a look at the rest of the changes and work on em tomorrow, thanks

Reply:
This should be fixed now, not sure why it didn't work last time.